user/rime/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "rime"
```

Comment:
the upstream repo is called librime, perhaps the package should be called the same